vagrant/lib/forklift/box_loader.rb

```diff
@@ -35,5 +35,15 @@ def versions
       YAML.load_file("#{@root_dir}/config/versions.yaml")
     end
 
+    def box_config
+      config = if File.exist?("#{@root_dir}/config/boxes.yaml")
+                 "#{@root_dir}/config/boxes.yaml"
+               else
+                 "#{@root_dir}/config/boxes.all.yaml"
```

Comment:
I'd like to avoid a `all` config because it's an additional thing to maintain. IMHO the code should handle when there's no config. The alternative is a special value for `operating_systems` and `foreman_versions` where instead of a list you can pass in the string `all`.